src/microbe_stage/editor/HexPopupMenu.cs

```diff
@@ -108,6 +108,8 @@ public virtual bool EnableModifyOption
         }
     }
 
+    private bool CanModifyHex => Visible && !Closing;
```

Comment:
This seems overall a sensible change, but I'm kind of unsure if this is the best name for this property. Would something like `IsInteractable` or maybe `IsOpenAndNotClosing` be a better name? As this also controls if delete is allowed and not just hex modification.

Also if this fix works, could you also check the multicellular metaball popup? It probably has the exact same bug as this class.

Reply:
The multicellular metaball popup derives from HexPopupMenu and doesn't add any new ways to accept input, so it did have this problem, but it's already fixed by the changes to HexMenuPopup.

Reply:
Ah that is good then.